### PR TITLE
prepare the SplunkEnterprise-SES-8.0.0-python38 job

### PR DESCRIPTION
**Pull request by goneri**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/471

Based on `ansible-network-appliance-base`.

### CODE REVIEW

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

File: zuul.d/project-templates.yaml [hunk 1]
```diff
@@ -839,6 +839,23 @@
...
               ansible_test_collections: true
 
+- project-template:
+    name: ansible-collections-security-splunk-es
+    check:
+      jobs:
+        - build-ansible-collection
+        - ansible-security-integration-splunk-es-python36:
+            vars:
+              ansible_test_collections: true
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:849]: this could be moved into the job stanza, as it only runs on collections now. No ansible/ansible stable branches

Comment (goneri) [reply zuul.d/project-templates.yaml]: Good idea.

**Pull request merged**